vimeo-networking/src/main/java/com/vimeo/networking/utils/ISO8601.java

```diff
@@ -98,9 +102,6 @@ public static Date toDate(final String iso8601string) throws ParseException {
         // Account for ISO 8601 standard of using the character 'Z' to represent UTC
         String s = iso8601string.replace("Z", "+00:00");
         try {
-            if (s.charAt(22) != ':') {
-                throw new ParseException("Invalid ISO 8601 format", 0);
-            }
```

Comment:
Well we throw a ParseException still, and SimpleDateFormat would throw a ParseException. In either case, nothing different than before - the app will crash unless the caller catches a ParseException.